project/build.scala

```diff
@@ -24,7 +24,16 @@ object build extends Build {
     packagedArtifacts := Map.empty,
     unidocProjectFilter in (ScalaUnidoc, unidoc) := inAnyProject,
     aggregate in test := false,
-    test := (test in scalameta in Test).value,
+    test := {
+      val runTests = (test in scalameta in Test).value
+      val runDocs = (run in readme in Compile).toTask(" --validate").value
+    },
+    publish := {
+      // Others projects are published automatically because we aggregate.
+      val publishDocs = (publish in readme).value
+    },
+    // TODO: The same thing for publishSigned doesn't work.
+    // SBT calls publishSigned on aggregated projects, but ignores everything else.
```

Comment:
/cc @eed3si9n. Maybe you'll be able to come up with something here. Here's the background that introduces the need for this: https://github.com/scalameta/scalameta/pull/421/files/5be17c403529e85d39cbac3c9f73723d1acac6b2#r65814536.